include/ui.js

```diff
@@ -84,8 +80,8 @@ var UI;
             UI.addSettingsHandlers();
 
             // Show the connect panel on first load unless autoconnecting
-            if (!autoconnect && !UI.connSettingsOpen)
-                UI.toggleConnectPanel();
+            if (!autoconnect)
```

Comment:
Given a choice I prefer to leave them out, but leaving them in is the dominant style here. I'll fix it.

PS. Is there a style guide somewhere for noVNC?